### DISCUSSION TITLE
[Feature request] No support for request time out while creating connection

### DISCUSSION THREAD

**Discussion by mohanraj-eee** [ing-bank/cassandra-jdbc-wrapper#5] [General]
Currently there is no support for "DefaultDriverOption.REQUEST_TIMEOUT" in driverConfigLoaderBuilder. Due to this I am getting an exception whenever default timeout 2S is exceeded. Is there a possibility to add request timeout while initiating connection?

**Comment by maximevw** [comment]
> Currently there is no support for "DefaultDriverOption.REQUEST_TIMEOUT" in driverConfigLoaderBuilder. Due to this I am getting an exception whenever default timeout 2S is exceeded. Is there a possibility to add request timeout while initiating connection? 

Hello @mohanraj-eee,
You're right, this is a good point. We'll add this option in the next release currently in preparation.

**Comment by maximevw** [comment]
This feature has been released in the version [4.6.0](https://github.com/ing-bank/cassandra-jdbc-wrapper/releases/tag/v4.6.0).
